redis_cache/rediscache.py

```diff
@@ -79,29 +78,40 @@ def __init__(self,
                  password=None,
                  ssl=False,
                  client=None,
-                 namespace="SimpleCache"):
+                 namespace="SimpleCache",
+                 read_only_host=None):
 
         self.limit = limit  # No of json encoded strings to cache
         self.expire = expire  # Time to keys to expire in seconds
         self.prefix = namespace
         self.host = host
         self.port = port
         self.db = db
+        self.password = password
+        self.ssl = ssl
+        self.read_only_host = read_only_host
 
         try:
             if client:
-                self.connection = client
+                self.write_connection = client
```

Comment:
can you also please add a read_only_client so that if we use this area, none of the variables are going to exist to make the read_connection object if we only pass in a redis client object.